src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -214,6 +216,11 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             response.Capabilities.Add("HandlesWorkerTerminateMessage", bool.TrueString);
             response.Capabilities.Add("HandlesInvocationCancelMessage", bool.TrueString);
 
+            if (workerOptions.IncludeEmptyEntriesInMessagePayload)
```

Comment:
If the option is disabled, we just do not send the capability at all (instead of sending with `False` value. Host code is not checking the value of this entry. It just checks for the existence of this capability key.

https://github.com/Azure/azure-functions-host/blob/ec3c2d1049351870bbf611760520cf1e48075c8f/src/WebJobs.Script.Grpc/MessageExtensions/GrpcMessageConversionExtensions.cs#L328-L331